test/dialing/direct.node.js

```diff
@@ -96,6 +96,14 @@ describe('Dialing (direct, TCP)', () => {
       .and.to.have.nested.property('._errors[0].code', ErrorCodes.ERR_TRANSPORT_UNAVAILABLE)
   })
 
+  it('should fail appropriately if peer has no addresses', async () => {
+    const dialer = new Dialer({ transportManager: localTM, peerStore })
+    const peer = new PeerId(Buffer.from('QmfEa1LrSt2R7KiZ31yPo3TT8ipfyDfrMRkaQix3rcXeTo'))
```

Comment:
Can you use the `createPeerId` function already required to create the `PeerId`? We try to use that because it creates a PeerId from a JSON which already have everything computed and the tests will be faster to run.

Reply:
k